src/full-promises.test.ts

```diff
@@ -1,7 +1,8 @@
-import { ClientSecurity } from "./util";
-import * as v1 from "./v1";
-import { Consistency } from "./v1";
-import { generateTestToken } from './__utils__/helpers' 
+import { ClientSecurity } from "./util.js";
+import * as v1 from "./v1.js";
+import { Consistency } from "./v1.js";
+import { generateTestToken } from './__utils__/helpers.js' 
```

Comment:
Typescript is sticky on this point - if you're using `node16` as the module target, you need to have a file suffix on your relative imports. Apparently this is a part of the JS spec that TS implements but it mostly annoys people.

I ran `tsc-esm-fix` to generate these changes.